src/theory/quantifiers/term_database.cpp

```diff
@@ -117,16 +117,9 @@ unsigned TermDb::getNumGroundTerms(Node f) const
 Node TermDb::getGroundTerm(Node f, unsigned i) const
 {
   std::map<Node, std::vector<Node> >::const_iterator it = d_op_map.find(f);
-  if (it != d_op_map.end())
-  {
-    Assert(i < it->second.size());
-    return it->second[i];
-  }
-  else
-  {
-    Assert(false);
-    return Node::null();
-  }
```

Comment:
Not sure if @ajreynol will like these changes. I already had a discussion in a review with Andy. These Assert(false) are ok to fail in a production build and the Engine can still continue (hence returning Node::null()). In a debug build, of course the assertion should fail.